### ISSUE TITLE
last_changed_seconds wrong due to timezone

### ISSUE THREAD

**Issue by sander1988** [AppDaemon/appdaemon#1571]
I noticed that the last_changed_seconds property of an entity always is 7200 wrong. My timezone is +02:00 so this has to be a timezone issue.

It looks like 'now' isn't in UTC: https://github.com/AppDaemon/appdaemon/blob/4.2.1/appdaemon/entity.py#L575

**I debugged the values during a state change trigger:**
- last_changed property of entity: 2022-09-17T12:39:58.203138+00:00
- 'now' value at that time: 2022-09-17 12:39:58.219077+02:00 - There is no 'T' here, that's not a typo from my side.
- last_changed_seconds value: -7199.982804

**Comment by sander1988** [comment]
I use this a current workaround, it might help someone fixing it:
```
    def _convert_to_seconds_ago(self, last_changed) -> float:
        utc = iso8601.parse_date(last_changed)
        now = datetime.datetime.now(datetime.timezone.utc)
        return (now - utc).total_seconds()
```

**Comment by Odianosen25** [comment]
Thanks for reporting this @sander1988, we will look into it.

kind regards

**Comment by acockburn** [comment]
This is fixed in 4.0.3
